Pract. 02/Task12.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+using namespace std;
+int main() {
+
+	int day, month;
+	cin >> day >> month;
+	bool isValid = true;
+
+
+
+	if (month == 1 || month == 3 || month == 5 || month == 7 || month == 8 || month == 10 || month == 12) {
+
+		if (day <= 30 && day > 1) {
+
+			day = day + 1;
+}
+		else if (day == 31 && day > 1) {
```

Comment:
You don't need to check if the day is bigger than 1, since it is either 31 or the if statement will be skipped.